presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -7859,6 +7863,26 @@ public void testQuantifiedComparison()
         assertQuery("SELECT cast(1 as decimal(3,2)) <> ANY(SELECT cast(1 as decimal(3,1)))");
     }
 
+    @Test(dataProvider = "quantified_comparisons_corner_cases")
+    public void testQuantifiedComparisonCornerCases(String query) {
+        assertQuery(query);
+    }
+
+    @DataProvider(name = "quantified_comparisons_corner_cases")
+    public Object[][] qualifiedComparisonsCornerCases() {
```

Comment:
Ideally, these tests should go with the commit that fixes the issue. Otherwise, the cause a build based on this commit to break, which can make it harder to bisect issues. It also makes it harder to "see" which specific scenarios are fixed.